multinet/user.py

```diff
@@ -137,3 +138,20 @@ def filtered_user(user: User) -> FilteredUser:
 def copy_user(user: User) -> User:
     """Create and return a new instance of User."""
     return from_dict(User, dataclasses.asdict(user))
+
+
+def search_user(query: str) -> Cursor:
+    """Search for users given a partial string."""
+
+    coll = user_collection()
+    aql = read_only_db("_system").aql
+
+    query = f"""
+        FOR doc in {coll.name}
+          FILTER CONTAINS(LOWER(doc.name), LOWER('{query}'))
+            OR CONTAINS(LOWER(doc.email), LOWER('{query}'))
+
+          RETURN doc
+    """
```

Comment:
> It might be that this is an enhancement, too, and that we could open a low priority issue for this

Let's pursue this as an enhancement. @JackWilb, can you file an issue and link it back to this comment thread?

Reply:
Opened #423